xcm/xcm-executor/src/config.rs

```diff
@@ -29,6 +29,9 @@ pub trait Config {
 	/// How to withdraw and deposit an asset.
 	type AssetTransactor: TransactAsset;
 
+	/// How to execute HRMP-related actions
+	type HrmpExecutor: ExecuteHrmp; 
```

Comment:
I think this could be improved by specifying what exactly actions are (i.e. the channel management). It is also better to take into account how the developers will approach it. I think this is mostly relevant for relay-chains which is a way less common case. Thus it would be good to guide a typical implementer of this to use the `()` dummy